test/pylib/rest_client.py

```diff
@@ -10,7 +10,7 @@
 from collections.abc import Mapping
 import logging
 import os.path
-from typing import Any, Optional
+from typing import Any, Dict, Optional
```

Comment:
nit: you can actually use `dict` in newer Python versions, which is a built-in so you don't have to import anything from `typing`.